rush.json

```diff
@@ -72,7 +72,7 @@
       "shouldPublish": true
     },
     {
-      "packageName": "@uifabric/demo",
+      "packageName": "@uifabric/fabric-website-resources",
       "projectFolder": "apps/demo",
```

Comment:
I think the folder name should mirror the package name to avoid confusion down the road..

Reply:
I'll rename folders in another PR - I wanted to minimize the risk with this PR to unblock